pkg/binding/controller.go

```diff
@@ -260,6 +301,23 @@ func (c *Controller) AppendKSResources(ctx context.Context) error {
 func (c *Controller) Start(parentCtx context.Context, workers int, cListers chan interface{}) error {
 	logger := klog.FromContext(parentCtx).WithName(controllerName)
 	ctx := klog.NewContext(parentCtx, logger)
+
+	// Create informer on managedclusters so we can re-evaluate BindingPolicies.
+	// This informer differ from the previous informers in that it listens on the ocm hub.
```

Comment:
It would be more accurate to replace `previous` with `other` since this is the first reference to an informer in this body's call.

It would also be better for the eye if the calls that refer to the other informers are preceded with a new line since otherwise it may look like the comment refers to the entirety of the block formed by the connected lines. This is an opinion and preference on this matter, not a fact.

Reply:
The force-push to ddcd3f666 makes these changes.